recipe/meta.yaml

```diff
@@ -27,8 +27,6 @@ requirements:
     - openssl 1.0.2r
```

Comment:
This should probably not pin openssl like this....

Reply:
It was intentional to force conda to build a binary against 1.0 in staged recipes. I'm using this for something that depends on `pygsi` and it doesn't support OpenSSL 1.1 (yet).

I'll migrate the various packages I did it in soon.